config/webpack/terra-dev-site.webpack.config.js

```diff
@@ -42,13 +43,36 @@ const aliasMonoRepoPackages = (monoRepo, hotReloading, webpackAliasOptions = {},
   }, {});
 };
 
+/**
+* Create index.html files for terra-dev-site and the additional apps.
+*/
+const indexPlugin = ({
```

Comment:
Should we export or move to it's own file if we are re-using in orion-dev-site?

Reply:
Possibly, Those changes will be made with orion-dev-site